templates/boilerplate-template/package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "move:test": "node ./scripts/move/test",
     "move:compile": "node ./scripts/move/compile",
```

Comment:
I wonder if we can get the ABI with compilation? instead of making a get request after publishing

Reply:
no it can only be downloaded from published contract